addons/l10n_in/models/account_invoice.py

```diff
@@ -84,7 +84,7 @@ def _post(self, soft=True):
         posted = super()._post(soft)
         gst_treatment_name_mapping = {k: v for k, v in
                              self._fields['l10n_in_gst_treatment']._description_selection(self.env)}
-        for move in posted.filtered(lambda m: m.country_code == 'IN'):
+        for move in posted.filtered(lambda m: m.country_code == 'IN' and m.move_type == 'out_invoice'):
```

Comment:
shouldn't we use the fiscal country code instead of the country code ?
Also, if those checks are only used for the EDI, they should preferably be put in `_check_move_configuration` I think, but anyway.
Why targeting master ? This line is also there in 15.0

Reply:
Hello @JulienVR 
Thanks for the review
I have updated the changes
and also I retarget the branch to 15.0
Please have a look once if free
Thanks

Reply:
ping @smetl
Could I have your opinion on this ? Do you think we should use the country_code or the fiscal_country_code ?

Reply:
Does not country_code already go to fiscal_country_code (which is on the company)

Reply:
@JulienVR That field is related to the fiscal country code.

Reply:
you're right, good then, thanks ! :sweat_smile: